jina/docker/hubio.py

```diff
@@ -336,7 +336,7 @@ def build(self) -> Dict:
 
             if is_build_success:
                 if self.args.test_uses:
-                    from jina.excepts import PretrainedModelFileDoesNotExist
+                    from jina.excepts import ModelCheckpointNotExist
```

Comment:
It is a problem, because a lot of `executors` in the hub will be able to be built again.

Reply:
sure, let me inherit it